mkdocs.yml

```diff
@@ -194,3 +194,8 @@ plugins:
             - docs/scripts/extensions.py:RuntimeBasesExtension
             - docs/scripts/extensions.py:SourceCodeExtension
             # - docs/scripts/extensions.py:ComponentSyntaxesExtension
+  - macros:
+      render_by_default: false
+      force_render_paths: community/people.md
```

Comment:
Just vocalizing for myself, so the combination of `render_by_default: false` and `force_render_paths` is so that we **opt-in** when we want to treat a page as a Jinja2 template. [See docs](https://mkdocs-macros-plugin.readthedocs.io/en/latest/rendering/#opt-in-through-the-config-file)